app/tests/api/sector.test.ts

```diff
@@ -0,0 +1,163 @@
+import { POST as createSector } from "@/app/api/v0/city/[city]/inventory/[year]/sector/route";
+import {
+  DELETE as deleteSector,
+  GET as findSector,
+  PATCH as updateSector,
+} from "@/app/api/v0/city/[city]/inventory/[year]/sector/[sector]/route";
+import { db } from "@/models";
+import { CreateSectorRequest } from "@/util/validation";
+import env from "@next/env";
+import assert from "node:assert";
+import { randomUUID } from "node:crypto";
+import { after, before, beforeEach, describe, it } from "node:test";
+
+import { createRequest } from "../helpers";
+
+import { Sector } from "@/models/Sector";
+
+const sectorId = randomUUID();
+const sectorName = "Test Sector";
+const locode = "XX7_INVENTORY_CITY";
```

Comment:
You could call this XX_SECTOR_CITY instead.

The idea is that you can find out which test suite has created the data and not cleaned up if it starts multiplying in the DB. Maybe we should start setting up a proper system for fixtures/ mock data for tests...